pkg/apis/machineremediation/v1alpha1/replicas.go

```diff
@@ -0,0 +1,35 @@
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package v1alpha1 contains API Schema definitions for the healthchecking v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=machineremediation.kubevirt.io
+package v1alpha1
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+        "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+        NamespaceOpenShiftMachineAPI = "openshift-machine-api"
+        MasterNodeRoleLabel = "node-role.kubernetes.io/master"
+)
+
+func GetReplicaCount(c client.Client) int32 { 
```

Comment:
IMHO it should return `(int32, error)` in case of an error we want just reconcile without changing anything